toolchain/scripts/start_arangoproxy.sh

```diff
@@ -32,4 +32,7 @@ if [ "$HUGO_ENV" != "prod" ] && [ "$HUGO_ENV" != "frontend" ]; then
   done
 fi
 
-cd /home/toolchain/arangoproxy/cmd && go run main.go $ARANGOPROXY_ARGS
+cd /home/toolchain/arangoproxy/cmd
+go mod vendor
+go build -o arangoproxy
```

Comment:
Do we need to update any documentation about this change?
In the README.md, we have the following, commented-out content, but I think we can delete it?

> #### Run without Docker
> 
> - Build and start the _arangoproxy_ web server
> 
>   ```shell
>   toolchain/arangoproxy/cmd> go build -o arangoproxy
>   toolchain/arangoproxy/cmd> ./arangoproxy {flags}
>   ```
> - Launch the hugo build command
> 
>   ```shell
>   docs-hugo/site> hugo
>   ```
> 
> The static HTML is placed under `site/public/`.
> 
> For development purpose, it is suggested to use the `hugo serve` command for
> hot-reload on changes. The runtime server is available at `http://localhost:1313/`.
> 
>

Reply:
@dandimeo can you please clarify?

Reply:
the local cached deps are saved in arangoproxy/vendor folder

Reply:
The "Run without Docker" instructions that @Simran-B posted above are commented-out. The question is whether we need this content or not. If it is applicable now, I guess we should keep them and uncomment them.

Reply:
In my opinion, I would get rid of the without docker option and let Docker as the only way to run the toolchain.

In case you want to have the option anyway, I can provide you more detailed steps to do it, as some other configuration has to be done in case.

That's up to you, in case I can open a new PR with the No Docker instructions

Reply:
I don't think we need a non-Docker way